gitlab-ci/src/bazel-ci/diff.sh

```diff
@@ -52,10 +52,11 @@ if [ ${#files[@]} -eq 0 ]; then
     exit 0
 fi
 
+UNIVERSE="$(echo ${BAZEL_TARGETS:-//...} | sed 's/ /+/')"
```

Comment:
What's `UNIVERSE` and why the substitution?

Reply:
Good question. I added some docs.